src/components/CallToAction/ExtendedGetInstanceCallToAction.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import { Button, TextField, Radio, Checkbox, TextLink as Link } from 'components';
+import { createUseStyles } from 'react-jss';
+
+import { FORM_NAMES } from '../../contactFormConstants';
+import { currentlyExecutingGitBranch } from '../../environment';
+
+const useStyles = createUseStyles(() => ({
+  fieldset: {
+    marginBottom: '2em',
+  },
+
+  radioWrapper: {
+    marginBottom: '0.4em',
+  },
+}));
+
+const SCM_TOOLS = [{
+  value: 'github-enterprise-cloud',
+  label: 'GitHub Enterprise (Cloud)',
+}, {
+  value: 'github-enterprise-on-prem',
+  label: 'GitHub Enterprise (On-prem)',
+}, {
+  value: 'github',
+  label: 'GitHub (Not sure)',
```

Comment:
Maybe 'standard' or something along those lines instead of 'Not sure'?

Reply:
@Xantier I was actually intending that option for people who think "I know we use GitHub but I'm not sure if it's the cloud version or the on-prem version". Thinking about it again, though... I have left out non-enterprise versions of GitHub, and I don't have a "Not sure" version of GitLab or Bitbucket.

Perhaps the best layout would instead be...

```js
const SCM_TOOLS = [{
  value: 'github-cloud',
  label: 'GitHub Cloud',
}, {
  value: 'github-on-prem',
  label: 'GitHub On-prem',
}, {
  value: 'gitlab-cloud',
  label: 'Gitlab Cloud',
}, {
  value: 'gitlab-on-prem',
  label: 'Gitlab On-prem',
}, {
  value: 'bitbucket-cloud',
  label: 'Bitbucket Cloud',
}, {
  value: 'bitbucket-server',
  label: 'Bitbucket Server',
}, {
  value: 'other',
  label: 'Other',
}];
```

Reply:
:+1: That makes sense to me

Reply:
I pushed a commit to switch to the setup described above.